domains/masseo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "matte-oss",
+    "email": "7sz8jlw6h@mozmail.co"
+  },
+  "record": {
+    "CNAME": "hashnode.network"
```

Comment:
I can not react your website, 404

can you provide a link to the website or show a preview of it?